Makefile

```
@@ -5,8 +5,21 @@ else
 endif
 OS := $(shell uname -s)
 
+# When src-tauri/modules/ contains staged binaries (e.g. aw-awatcher, aw-sync
+# prepared by the activitywatch bundle build), inject them via bundle.resources
+# so deb/rpm/AppImage are self-contained. Standalone builds without that dir are
+# unchanged. See ActivityWatch/aw-tauri#232.
 build: prebuild
```

checkmake (0.3.2)

[warning] Target "build" should be declared PHONY.

(phonydeclared)